ast/builtins.go

```diff
@@ -2289,6 +2290,17 @@ var CryptoX509ParseCertificateRequest = &Builtin{
 	),
 }
 
+var CryptoX509ParseKeyPair = &Builtin{
+	Name:        "crypto.x509.parse_keypair",
+	Description: "Returns a valid key pair",
+	Decl: types.NewFunction(
+		types.Args(
+			types.Named("cert", types.S).Description("string containing PEM or base64 encoded DER certificates"),
+			types.Named("pem", types.S).Description("string containing PEM or base64 encoded private key"),
+		),
+		types.Named("output", types.NewObject(nil, types.NewDynamicProperty(types.S, types.A))).Description("Keypair as an object"),
+	),
```

Comment:
It would be helpful to have a detailed description of the builtin output. Does `Keypair as an object` mean `X.509 Keypair as an object`? Some details would be good.

Reply:
@ashutosh-narkar  as it returns https://pkg.go.dev/crypto/tls#Certificate, is it sufficient to link to that?

Reply:
I would recommend adding some relevant text and not add a link in the builtin description.

Reply:
right. I've added a (hopefully) helpful text. see 911e7218aff3c3ae9dde6d8b5ddcb01f3d822f76